### PR TITLE
Remove howdiditstart validate 2221

### LINKED ISSUE #2221: The fields in How did the suspect reach you? page should be optional.
## Summary

In the current prod version, the fields in "How did the suspect reach you?" page is optional. but now it is changed. The user will get an error message "howDidTheyReachYou is a required field" if the user does not choose any field. If we decide the user has to choose at least one field in this page, the error message should be changed to make it more clear

## Steps to reproduce
1. Choose a full report
2. Without choosing any field in "How did the suspect reach you?" page, click on continue

page1

### PR DESCRIPTION
**Pull request by kevinyang06**
Fixes #2221

# Description

> Reopening issue. Validation should be removed from this page entirely. It was only added as an example for Formik (see PR #2138 ) but needs to be removed before we go to prod.

Issue #2138: "Note that the validation for this form is for testing purposes only. There are no validation rules defined for this form in production so I created one to test/demonstrate."

# Any new packages installed?

> Give details

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

**Comment by ying-chen-cds** [comment]
Is this PR related to Issue 2221?

**Comment by kevinyang06** [comment]
> Is this PR related to Issue 2221?

Yes, the new task is to remove validation.

File: f2/src/forms/__tests__/HowDidItStartForm.test.js [hunk 1]
```diff
@@ -71,6 +71,6 @@ describe('<HowDidItStartForm />', () => {
       await wait(0) // Wait for promises to resolve
     })
 
-    expect(submitMock).toHaveBeenCalledTimes(0)
+    expect(submitMock).toHaveBeenCalledTimes(1)
```
Comment (ying-chen-cds) [inline f2/src/forms/__tests__/HowDidItStartForm.test.js:74]: The name of this test "does not call the onSubmit function when the form is submitted" should be changed to "call the onSubmit function when the form is submitted"

File: f2/src/forms/HowDidItStartFormSchema.js [hunk 2]
```diff
@@ -1,12 +1,7 @@
 import * as Yup from 'yup'
-import React from 'react'
-import { Trans } from '@lingui/macro'
 import { yupSchema } from '../utils/yupSchema'
 
 const howDidItStartFormSchema = Yup.object().shape({
-  howDidTheyReachYou: Yup.array().required(
-    <Trans id="howDidTheyReachYou.error" />,
-  ),
   email: yupSchema().emailSchema,
```
Comment (justinr86) [inline f2/src/forms/HowDidItStartFormSchema.js:5]: If the goal is to remove validation shouldn't the entire Yup schema be removed?

**Review by justinr86 — APPROVED** [review]

**Pull request merged**